_articles/send/send-encryption.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: article
+title: Send Encryption
+categories: [send]
+featured: true
+popular: false
+tags: [bitwarden send, send, about send, ephemeral sharing]
+order: 07
+---
+
+## Send Encryption
+
+Sends are a secure and ephemeral mechanism for transmitting sensitive information to anyone, include plaintext and files. As the [About Send]({% link _articles/send/about-send.md %}) article notes, Sends are **end-to-end encrypted**, meaning that encryption (*described below*) and decryption occur client-side. When you create a Send:
+
+1. A new 16-byte secret key is generated for the Send.
```

Comment:
```suggestion
1. A new 128-bit secret key is generated for the Send.
```

Reply:
not sure if we use bytes or bits elsewhere in docs. I don't really care as long as we are consistent. sometimes "bit" sounds better since it's a bigger number.

Reply:
Looks like "bits" are more prevalent than "bytes" in the docs, so I went ahead with this change.